06-building-a-program-start-to-finish/writing-tests/Output/inverted-index-spec.js

```diff
@@ -0,0 +1,171 @@
+/* global describe, it, expect */
+const valid1 = require('../../src/test books/valid1.json'),
+      valid2 = require('../../src/test books/valid2.json'),
+      valid3 = require('../../src/test books/valid3.json'),
+      valid4 = require('../../src/test books/valid4.json'),
+      invalid1 = require('../../src/test books/invalid1.json'),
+      invalid2 = require('../../src/test books/invalid2.json'),
+      invalid3 = require('../../src/test books/invalid3.json'),
+      invalid4 = require('../../src/test books/invalid4.json'),
+      invalid5 = require('../../src/test books/invalid5.json'),
+      invalid6 = require('../../src/test books/invalid6.json'),
+      invalid7 = require('../../src/test books/invalid7.json');
+
+const valid3AsIndexed = { '777': [ 1 ], the: [ 0, 1 ], triumphant: [ 0 ],
+                          church: [ 0 ], christians: [ 0 ], are: [ 0 ],
+                          already: [ 0 ], victorious: [ 0 ], not: [ 0 ],
+                          militant: [ 0 ], or: [ 0 ], defeated: [ 0 ],
+                          written: [ 1 ], by: [ 1 ], a: [ 1 ], man: [ 1 ],
+                          considered: [ 1 ], most: [ 1 ], wicked: [ 1 ],
+                          that: [ 1 ], ever: [ 1 ], lived: [ 1 ],
+                          allister: [ 1 ], crowley: [ 1 ]
+                        }
+
+const valid4AsIndexed = { testing: [ 0 ], could: [ 0, 1 ], be: [ 0, 1 ],
+                          annoying: [ 0 ], when: [ 1 ], tests: [ 1 ],
+                          pass: [ 1 ], fulfilling: [ 1 ], and: [ 1 ],
+                          rewarding: [ 1 ] }
+
+const textString = 'why are We *% \n so blind \t to see two-timers#@!`~-_+=/?";';
+const tokenizedTextString = [ 'why', 'are', 'we', 'so', 'blind', 'to', 'see',
+                              'two', 'timers' ];
+
```

Comment:
I believe there is a better way of doing this. Wouldn't putting all of this into a file and then referencing it seem neater?